frontend/public/components/monitoring/dashboards/index.tsx

```diff
@@ -30,53 +31,128 @@ const evaluateTemplate = (s: string, variables: VariablesMap) =>
   _.reduce(
     variables,
     (result: string, v: Variable, k: string): string => {
-      return result.replace(new RegExp(`\\$${k}`, 'g'), v.value);
+      return result.replace(new RegExp(`\\$${k}`, 'g'), v.value === undefined ? '' : v.value);
```

Comment:
Assuming `null` should also be `''`, we could use

```suggestion
      return result.replace(new RegExp(`\\$${k}`, 'g'), v.value || '');
```

Reply:
👍 Thanks. `v.value` is a string so we don't need to worry about `0` values, etc.

Changed by #4221